mobile/status.go

```diff
@@ -99,6 +99,7 @@ func initializeApplication(requestJSON string) string {
 	return string(data)
 }
 
+// Deprecated: Use InitializeApplication instead.
 func OpenAccounts(datadir string) string {
```

Comment:
Nothing new here, it was deprecated already. It's just done for the private func:
https://github.com/status-im/status-go/blob/b3143ce2885c90559ac3dd7c1825df4f1f86f1c4/mobile/status.go#L107-L109